docs/users/molecule_cookbook.ipynb

```diff
@@ -167,7 +167,7 @@
    "id": "821c4687",
    "metadata": {},
    "source": [
-    "We can downgrade this error to a warning with the `allow_undefined_stereo` argument. This will not apply an improper dihedral term to the stereocenter and may lead to simulations with unphysical stereoisomerisation."
+    "We can downgrade this error to a warning with the `allow_undefined_stereo` argument. This will create a molecule with undefined chirality, which might lead to an incorrect parametrization for force fields that depend on stereochemistry. Note that main-line OpenFF force fields currently use a stereochemistry-dependent model for assigning charges. Generating coordinates for a chiral center with undefined stereochemistry might lead to any combination of stereoisomers; the Toolkit makes no guarantees about randomness or uniformity."
```

Comment:
I'm kind of with Matt; I was initially surprised by this answer, because I was thinking "... but our force fields don't (currently) depend on stereochemistry!" but then I realized you're right about electrostatics. So maybe both shoulud be mentioned?

Otherwise this PR looks good to me.

Reply:
Ahh, that's a good question. Overall I disagree with the amount of detail here -  I feel fairly strongly that this level of description shouldn't be in a cookbook, but we could add this to the FAQ and link to it. 

Regarding which parameters may be stereo-dependent - While I know that we stripped stereomarks out of the ff14sb port (and they were never present in main-line FFs), I'm not sure whether that will _always_ be the case. In fact, I can think of use cases for bespokefit where it WOULD need to distinguish which parameter to use based on stereochemistry. So this statement may change in the future and we shouldn't need to track down everywhere that we mention stereo to fix it - I'd prefer that this just link to a more thorough description in the FAQ.

Reply:
I think I agree with Jeff - this warning is already getting very long for a cookbook, and the FAQ could really use this level of detail (and already has a question for it!)

Reply:
Okay I've pushed the changes Jeff and I talked about in our meeting - I'll merge this tomorrow if no-one has any objections :)